### PR TITLE
Adding m_ for member variables in new diags

### PR DESCRIPTION
**Pull request by RevathiJambunathan**
This is a cleanup PR to add the prefix m_ for member variables defined in Diagnostics.H.

### CODE REVIEW

File: Source/Diagnostics/ComputeDiagFunctors/DivBFunctor.cpp [hunk 1]
```diff
@@ -12,7 +12,7 @@ DivBFunctor::operator()(amrex::MultiFab& mf_dst, int dcomp) const
 {
     auto& warpx = WarpX::GetInstance();
     // Guard cell is set to 1 for generality. However, for a cell-centered
-    // output Multifab, mf_avg, the guard-cell data is not needed especially considering
+    // output Multifab, m_mf_output, the guard-cell data is not needed especially considering
```
Comment (MaxThevenet) [inline Source/Diagnostics/ComputeDiagFunctors/DivBFunctor.cpp:15]: Shouldn't't this actually be `mf_dst`?

File: Source/Diagnostics/ComputeDiagFunctors/DivEFunctor.cpp [hunk 2]
```diff
@@ -12,7 +12,7 @@ DivEFunctor::operator()(amrex::MultiFab& mf_dst, const int dcomp) const
 {
     auto& warpx = WarpX::GetInstance();
     // Guard cell is set to 1 for generality. However, for a cell-centered
-    // output Multifab, mf_avg, the guard-cell data is not needed especially considering
+    // output Multifab, m_mf_output, the guard-cell data is not needed especially considering
```
Comment (MaxThevenet) [inline Source/Diagnostics/ComputeDiagFunctors/DivEFunctor.cpp:15]: Same comments, which also holds in other files below.

File: Source/Diagnostics/ComputeDiagFunctors/DivBFunctor.cpp (hunk 1 above)
Comment (RevathiJambunathan) [reply Source/Diagnostics/ComputeDiagFunctors/DivBFunctor.cpp:15]: That is right! Changed it now. Thank you!!

File: Source/Diagnostics/ComputeDiagFunctors/DivEFunctor.cpp (hunk 2 above)
Comment (RevathiJambunathan) [reply Source/Diagnostics/ComputeDiagFunctors/DivEFunctor.cpp:15]: Made this modification to mf_dst as well.

**Pull request merged**